homeassistant/components/utility_meter/strings.json

```diff
@@ -0,0 +1,29 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "New Utility Meter",
+        "description": "Meter reset offset allows offsetting the time of meter reset.\nSupported tariffs is a comma separated list of supported tariffs, leave empty if only a single tariff is needed.",
```

Comment:
This needs to actually contain a description of what a utility meter is?

Reply:
Yeah, and I think we should have that in all the helper flows.

In this case, something like this maybe:
> The utility meter sensor provides functionality to track consumptions of various utilities (e.g., energy, gas, water, heating) over a configured period of time, typically monthly. The utility meter sensor also supports splitting the consumption by tariffs.